### PR TITLE
Minor corrections for showing surfaces 

### PR DESCRIPTION
**Pull request by Garyfallidis**
This PR updates what was merged with PR https://github.com/nipy/dipy/pull/1034
I also rendered the tutorial which outputs correctly a simple cube

image

@MarcCote and @ranveeraggarwal can you look at it and merge it?

### CODE REVIEW

File: doc/examples/viz_surfaces.py [hunk 1]
```diff
@@ -1,14 +1,20 @@
 """
-========================================
-Visualize surfaces; load/save, get/set and update vtkPolyData.
-========================================
+==================
+Visualize surfaces
+==================
+
+Here is a simple tutorial that shows how to visualize surfaces using DIPY. It shows as well how to load/save, get/set and update vtkPolyData and show
```
Comment (arokem) [inline doc/examples/viz_surfaces.py]: PEP8

File: doc/examples/viz_surfaces.py [hunk 2]
```diff
@@ -1,14 +1,20 @@
...
+surfaces.
 
-import usefull functions and dipy utils
+vtkPolyData is a structure used by VTK to represent surfaces and other data structures. Here we show how to visualize a simple cube but the same idea should apply for any surface.
```
Comment (arokem) [inline doc/examples/viz_surfaces.py]: Same here (too long)

File: doc/examples/viz_surfaces.py (hunk 1 above)
Comment (arokem) [inline doc/examples/viz_surfaces.py]: "It shows as well" => "It also shows"

File: doc/examples/viz_surfaces.py [hunk 3]
```diff
@@ -1,14 +1,20 @@
...
 """
-from __future__ import division, print_function, absolute_import
+
 import numpy as np
 
-# and dipy tools
+"""
+Import usefull functions from dipy.viz.utils
```
Comment (arokem) [inline doc/examples/viz_surfaces.py]: typo: "usefull" => "useful"

**Comment by MarcCote** [comment]
LGTM. It could be done in another PR but I agree with @arokem showing a brain surface would be nice. Do we have any brain surfaces saved in `.vtk` accessible from the dipy.data fetcher?

**Comment by arokem** [comment]
No brain surfaces accessible through the fetchers... yet. We'd need to create that.

**Comment by Garyfallidis** [comment]
Yes, absolutely. The idea is to have a brain surface here. I asked @StongeEtienne to add a fetcher for a brain surface from some public datasets. Should be easy to do. I think he can do this early next week. If you guys have already already a brain surface that is even better go ahead and create a fetcher. However, if you have a brain surface that is in the same space as data from other fetchers that we already have that would be fantastic! In the mean time I think we can merge this PR.

**Comment by Garyfallidis** [comment]
Finally, on Ariel's question about showing together surfaces and streamlines. I think that should probably go to another tutorial or to the existing advanced viz tutorial which will be updated after @ranveeraggarwal and @MarcCote 's PRs are merged.

**Comment by Garyfallidis** [comment]
@arokem is there anything else?

**Comment by arokem** [comment]
We can probably pretty easily make a brain surface for the Stanford HARDI data-set. I'll get cracking on it.

**Comment by Garyfallidis** [comment]
Okay but you will do that in another PR right?

**Comment by arokem** [comment]
Sure. Different PR. 

This seems fine to me. If no one else has any comments, I can merge it early next week.

**Comment by Garyfallidis** [comment]
I think @MarcCote can merge this one today if he has time. There are only documentation changes here. Agreed @arokem? Sorry, we are trying to push on with the visualization this week.

**Pull request merged**